src/cljs/rems/application.cljs

```diff
@@ -855,11 +1001,15 @@
     [:div.col-lg-8
      [application-state application config highlight-request-id]
      [:div.mt-3 [applicants-info application]]
+     (when (:enable-duo config)
+       (if (= userid (-> application :application/applicant :userid))
```

Comment:
What about members? What kind of DUO view shoudl they have?

Reply:
i don't know. what does a member do? is member that person who is invited to an application?

Reply:
added check for `:member` role